lib/templates-sync.php

```diff
@@ -86,12 +89,27 @@ function _gutenberg_synchronize_theme_templates( $template_type ) {
 		'template'      => 'block-templates',
 		'template-part' => 'block-template-parts',
 	);
+	$themes              = array_unique( array( get_stylesheet(), get_template() ) );
```

Comment:
What about using this:

```php
$themes = array_unique( array( get_stylesheet_directory(), get_template_directory() ) );
```

Then we could avoid the call to `wp_get_theme` and `get_stylesheet_directory`. Then we could just use the directory as the slug identifier, or just the basename of the patth.

Probably not a big deal either way, given that we're doing a lot of filesystem stuff here anyways.

Reply:
`basename` wouldn't return correct `stylesheet` in case of "nested" themes. ( for example themes/seedlet-blocks would become seedlet-blocks)

I used an assoc array though to avoid creating WP_Theme objects.